src/lib/testPlanGenerator.ts

```diff
@@ -0,0 +1,36 @@
+import { exportTestPlan as exportTestPlanToCsv } from "./testPlanFormatter";
+import { loadFeaturesFrom } from "./featureProcessor";
+import { TestCase, Feature } from "./testPlan";
+
+/**
+ * Traverse a feature file directory, collect test cases
+ * and write test plan template to csv file
+ * @param features
+ */
+export async function generateTemplate(
+   featureDirectory: string,
+   filename: string,
+   versionToTest: string
+): Promise<string> {
+   const features = await loadFeaturesFrom(featureDirectory);
+   const testCases = generateTestCases(features);
+   return await exportTestPlanToCsv({
```

Comment:
looks like you can remove `await` here since return type is a promise.

Reply:
I'd vote to keep the consistency of awaiting a promise where ever it is used - I think it makes it easier to refactor because you don't have to think about if the thing that is returned is a promise or not

Reply:
Done.  Overzealous promise resolutions...